cypress/utils/utils.ts

```diff
@@ -29,6 +29,7 @@ import {
     userPerspectiveMenu,
 } from "../integration/types/constants";
 import { actionButton, date } from "../integration/views/applicationinventory.view";
+import { dropdownMenuToggle } from "../integration/views/tags.view";
```

Comment:
Is this required ? I do not see it being used

Reply:
yes, I will remove it, one sec